src/hooks/usePublicMessages.js

```diff
@@ -47,14 +49,14 @@ export default function usePublicMessages(infoPoint, routes) {
 
   return useMemo(() => {
     if (!(publicMessages instanceof Array)) return publicMessages;
-    return publicMessages.sort(comparePublicMessages).filter((publicMessage) => {
-      if ((routes instanceof Array) && (publicMessage.routeAbbreviation)) {
-        return routes.includes(publicMessage.routeAbbreviation);
+    return publicMessages.filter((publicMessage) => {
+      if (routes instanceof Array) {
+        return (publicMessage.routes.filter((route) => {
+          return routes.includes(route.abbreviation) || route === null;
```

Comment:
I'm a little confused about the `route === null` condition - in what case will that come up?

Reply:
I think this was leftover from an earlier implementation/I got confused about how routes would get read? But there's no case from what I can see now